package.json

```diff
@@ -23,19 +23,18 @@
   ],
```

Comment:
Github doesn't allow for adding comments on collapsed/expanded lines, so this comment is meant for line 13 above:

If this is being published to NPM, you might want to change that repository "url" property to point to this repo.

Reply:
Yeah I'm gonna have to fix this up and figure out the publishing steps, as well as figuring out what version this becomes.